tests/dragonfly/replication_test.py

```diff
@@ -410,50 +410,41 @@ async def test_rotating_masters(
 @pytest.mark.slow
 async def test_cancel_replication_immediately(df_local_factory, df_seeder_factory):
```

Comment:
note that this test is not running in the CI because it is marked as slow

Reply:
I know, I will run it manually again and again and again :P